### PR TITLE
New `upgrade` command

### PR DESCRIPTION
**Pull request by afbora**

### CODE REVIEW

**Review by bastianallgeier — APPROVED** [review]
I made some small adjustments, but this is really cool to use! I love it.

File: commands/upgrade.php [hunk 1]
```diff
@@ -25,38 +26,41 @@
...
 
+		// checks if the current kirby version is the same as the new one
+		if (version_compare($version, $kirby->version(), '==') === true) {
+			$cli->success('Your Kirby installation is already up to date');
+			exit;
+		}
+
 		// checks current kirby version whether same or higher
 		if (version_compare($version, $kirby->version(), '<=') === true) {
 			throw new Exception('Current Kirby version is the same or higher than the version you are trying to upgrade to');
```
Comment (afbora) [inline commands/upgrade.php:37]: @bastianallgeier You added up-to-date message. So wee need to change this compare operator and message.

Comment (bastianallgeier) [reply commands/upgrade.php:37]: I already added the comparing operator. Not sure I understand what you mean.

Comment (afbora) [reply commands/upgrade.php:37]: If kirby version up-to-date (above new your lines), this checks should be like following. For ex case; current version 4.1.1 but version argument 4.0.0.

````php
// checks current kirby version and prevents downgrade
if (version_compare($kirby->version(), $version, '>') === true) {
    throw new Exception('Current Kirby version is higher than the version you are trying to upgrade to');
}
````

Comment (bastianallgeier) [reply commands/upgrade.php:37]: Ah, now I get it. I fixed it on develop

Comment (afbora) [reply commands/upgrade.php:37]: Thank you. Also updating message would be great. Because can't be same version since your new check lines 🙈

Comment (afbora) [reply commands/upgrade.php:37]: ### Current message
Current Kirby version is the same or higher than the version you are trying to upgrade to

### Possible
Current Kirby version is higher than the version you are trying to upgrade to

Comment (afbora) [reply commands/upgrade.php:37]: Done in develop branch https://github.com/getkirby/cli/commit/b86089f9d4b42119a883a029bd1767b33ef1b185

Comment (bastianallgeier) [reply commands/upgrade.php:37]: You are right! I made a small adjustment to your text. Now it looks good to me.

**Pull request merged**